lib/vmpooler/pool_manager.rb

```diff
@@ -705,6 +705,8 @@ def execute!
 
       # Clear out the tasks manager, as we don't know about any tasks at this point
       $redis.set('vmpooler__tasks__clone', 0)
+      # Clear out vmpooler__migrations since stale entries may be left after a restart
+      $redis.del('vmpooler__migration')
```

Comment:
Could you quickly explain the difference between clearing out the tasks manager and clearing out migrations? I'm mostly just curious why one has a value set to 0 to clear it out and the other is deleted completely.